tests/testthat/test_model_readinsharewts.R

```diff
@@ -0,0 +1,48 @@
+context("Model results with read in share weights")
+
+basepath <- file.path(tempdir(), "outputs")
+test.info <- update_scen_info(aName = "ReadInShWt", aCalibrateShareWt = FALSE)
+test.info$mOutputDir <- basepath
+
+test_that("Model runs successfully with read in share weights.", {
+    ## Run the model to generate outputs.  This needs to be the first test in
+    ## this context, as all the rest will depend on these results.
+    expect_message(run_model(test.info, aVerbose=TRUE))
+})
+
+
+test_that("shares for all nodes add to 1 with read in share weights", {
+  # Finally, test (NB rounding numeric columns to a sensible number of
+  # digits; otherwise spurious mismatches occur)
+  # Also first converts integer columns to numeric (otherwise test will
+  # fail when comparing <int> and <dbl> columns)
+  DIGITS <- 1
```

Comment:
Same comments as above, re number of digits and use of `tolerance` in `expect_equivalent`.